### ISSUE TITLE
Adding to Unraid community apps store?

### ISSUE THREAD

**Issue by bgriffen** [calebolson123/BabySleepCoach#29]
Just wondering if you thought of maybe [adding to the community apps of Unraid](https://unraid.net/community/apps) because will definitely help adoption! If it really is the case of just providing the web feed, I know at least 5 people who could use this immediately. They just aren't tech savvy enough for rasperrypi. One quick way might be to add the image to docker-hub.

Great project!

**Comment by djuniah** [comment]
Been trying to get this to run on unraid on and off for the past few weeks.  Got it building properly using the docker-compose plugin after some tweaking, however I can't seem to figure out network access.  It's definitely running but i can't access the web GUI.  Probably some small config thing that I'm missing.  Either way, I'm pretty sure that unraid community apps have to be pre-built hosted containers on services like dockerhub and not compose-based ones, but i could be wrong about that.

**Comment by calebolson123** [comment]
Huh, first time learning about the unraid community/store. I used docker-compose because it was most familiar and easy to get running. A slight upgrade over raw web and server.. but throwing it all into a single image like you're describing wouldn't be that difficult.

Maybe I'll get around to doing this, most of the github issues on this repo are around setup/configuration, which is what I spent the least amount of time on! To do the project justice, I really should make it easier for new users

**Comment by djuniah** [comment]
Yeah unraid itself is a pretty fantastic OS choice for a NAS build, but the "apps" that it has are just docker images with associated template files which help provide a nice GUI interface to the necessary parameters that a container needs to run.  As far as i'm aware, compose isn't really something that it handles natively through those templates.  There are plugins to allow users to use compose files, which is what I had done, but it's not included by default.  Making one image with everything in it would greatly simplify that process.  UnRaid can also search/pull directly from dockerhub or git's version of dockerhub(I can never remember the name), so as long as the image is available there, it "should" show up in the unraid app store.

**Comment by McCloudS** [comment]
I forked the repo and pushed the image to dockerhub @ mccloud/babysleepcoach

It's running on my unraid, see attached for example config.  
![image](https://github.com/calebolson123/BabySleepCoach/assets/64094529/488e4f99-26cd-426e-8726-35781191f2a7)
![image](https://github.com/calebolson123/BabySleepCoach/assets/64094529/81dd7bc1-6e9e-4b9c-93ad-e467f5759a56)
